legiti/src/test/java/com/legiti/UnitTests.kt

```diff
@@ -58,6 +61,22 @@ class UnitTests {
         assertTrue(LegitiConfig.isValid(authToken))
     }
 
+    @Test
+    fun testSetupWithInternalAuthToken() {
+        val config = LegitiConfig(AUTH_TOKEN)
+
+        assertEquals(AUTH_TOKEN.removePrefix("internal_"), config.authToken)
+        assertTrue(config.legitiDevEnv)
```

Comment:
not sure i understand what these are testing?

Reply:
I am testing that if I send something with the `internal_` prefix if the class will set the `legitiDevEnv` as true (meaning request will be going to staging) and if it removed the `internal_` from the authToken (since if we send something like `internal_sandboc_(...)` it will be rejected by heimdall)